### PR TITLE
Hybris 18.1

### PR DESCRIPTION
**Pull request by b100dian**
This should not be merged, but a hybris-18.1 branch should be made for it

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -297,6 +297,7 @@ bool EncryptOrDecryptOnce(bool encrypt,
   // finish
   result =
 	keystore->finishOperation(handle, empty_params,
+          std::string(), /* allow input data on finish */
```
Comment (rinigus) [inline main.cpp]: Formatting is off

File: main.cpp [hunk 2]
```diff
@@ -544,6 +545,7 @@ int SignKeyGen(const std::string& key_name)
...
```
Comment (rinigus) [inline main.cpp]: Formatting is off

**Review by rinigus — CHANGES_REQUESTED** [review]
I presume branch will be then android-11 . Or is there a reason for having it hybris-18.1?

Re formatting: probably tab/space issue

**Comment by b100dian** [comment]
> I presume branch will be then android-11 . Or is there a reason for having it hybris-18.1?

Not really - just because it is not mainline I thought about hybris. android-11 would do just fine

> Re formatting: probably tab/space issue

You're right, that wasn't visibile in my editor. Should be fixed now

**Comment by rinigus** [comment]
Thank you very much. Sorry for delay - android-11 branch has your changes now. Closing here

**Pull request closed without merging**